README.md

```diff
@@ -1,6 +1,8 @@
 WildFly Elytron Tool
 ====================
 
+WildFly Elytron's wildfly-elytron-tool repository is no longer in use to date, and it is reccomended to view wildfly-elytron/tool for relevant code for the Elytron Tool. 
```

Comment:
Just a small typo: "reccomended" -> "recommended"

It would be good to add a link to the new location:

"view wildfly-elytron/tool" -> "view https://github.com/wildfly-security/wildfly-elytron-tool"

Reply:
okay I will fix that, thanks

Reply:
Maybe we could reword to something like this:

WildFly Elytron's wildfly-elytron-tool repository is no longer in use to date. The code has moved to a new location: https://github.com/wildfly-security/wildfly-elytron/tree/1.x/tool

Reply:
sure we can I will do that